src/Middleware/ConcurrencyLimiter/sample/Startup.cs

```diff
@@ -17,19 +17,13 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddLIFOQueue((options) => {
-                options.MaxConcurrentRequests = Environment.ProcessorCount;
-                options.RequestQueueLimit = 50;
-            });
         }
 
         public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
-            app.UseConcurrencyLimiter();
```

Comment:
Shouldn't the sample still include this?

Reply:
ah yes